src/CloudNative.CloudEvents.NewtonsoftJson/JsonEventFormatter.cs

```diff
@@ -136,6 +136,44 @@ public override CloudEvent DecodeStructuredModeMessage(Stream data, ContentType
         public override CloudEvent DecodeStructuredModeMessage(byte[] data, ContentType contentType, IEnumerable<CloudEventAttribute> extensionAttributes) =>
             DecodeStructuredModeMessage(new MemoryStream(data), contentType, extensionAttributes);
 
+        public override async Task<IReadOnlyList<CloudEvent>> DecodeBatchModeMessageAsync(Stream data, ContentType contentType, IEnumerable<CloudEventAttribute> extensionAttributes)
```

Comment:
Consider updating to mention new methods https://github.com/cloudevents/sdk-csharp/pull/111/files#diff-b73904f7e0560631ddf370286e4d19cd3edf58de51d76bc28a6d33b11edb5b94R59

Reply:
Yes indeed! Thanks for the reminder. Will definitely do that.